### ISSUE TITLE
vkconfig: problem opening native file manager

### ISSUE THREAD

**Issue by johnzupin** [LunarG/VulkanTools#1464]
There seems to be a problem when vkconfig tries to open a native file manager on Linux. Vkconfig will just hang when it tries to open the file manager.

Environment:
- Ubuntu 20.04 (gnome or xfce)
- Nvidia

Ways to reproduce:
1)
    Choose the "Portability" layer configuration
    On the right panel, choose "..." to bring up file manager to select the DevSim json configuration file
2)
    Choose: "Edit..." after checking "Apply only to the selected list of applications"
    Without changing anything choose "Add"
3) 
    Export any layer configuration

**Comment by johnzupin** [comment]
I've bisected to this commit: 
```
e1af03a084e593031a3be469761a02b4923cd464 is the first bad commit
commit e1af03a084e593031a3be469761a02b4923cd464
Author: Christophe <christophe@lunarg.com>
Date:   Sat Apr 17 17:53:12 2021 +0200

    vkconfig: refactor validation panel
    
    Change-Id: Ia253d0b224fe2cc5940a5f9d1e4227586d4be435

 vkconfig/settings_tree.cpp               |   33 +-
 vkconfig/settings_tree.h                 |    4 +-
 vkconfig/settings_validation_areas.cpp   | 1077 +++++++++++++++++++-----------
 vkconfig/settings_validation_areas.h     |  168 +++--
 vkconfig/widget_setting_bool.cpp         |    9 +-
 vkconfig/widget_setting_bool.h           |    2 +-
 vkconfig/widget_setting_enum.cpp         |    7 +-
 vkconfig/widget_setting_enum.h           |    2 +-
 vkconfig/widget_setting_filesystem.cpp   |   31 +-
 vkconfig/widget_setting_filesystem.h     |    3 +-
 vkconfig/widget_setting_flags.cpp        |    7 +-
 vkconfig/widget_setting_flags.h          |    2 +-
 vkconfig/widget_setting_float.cpp        |    7 +-
 vkconfig/widget_setting_float.h          |    2 +-
 vkconfig/widget_setting_frames.cpp       |    7 +-
 vkconfig/widget_setting_frames.h         |    2 +-
 vkconfig/widget_setting_int.cpp          |    7 +-
 vkconfig/widget_setting_int.h            |    2 +-
 vkconfig/widget_setting_list.cpp         |   16 +-
 vkconfig/widget_setting_list.h           |    4 +-
 vkconfig/widget_setting_list_element.cpp |    7 +-
 vkconfig/widget_setting_list_element.h   |    2 +-
 vkconfig/widget_setting_string.cpp       |    7 +-
 vkconfig/widget_setting_string.h         |    2 +-
 vkconfig_core/setting_meta.cpp           |   10 +
 25 files changed, 897 insertions(+), 523 deletions(-)
```

**Comment by christophe-lunarg** [comment]
This issue should be fixed with https://github.com/LunarG/VulkanTools/pull/1463.

**Comment by johnzupin** [comment]
Can confirm fixed as of ref: v1.2.176-9-g125ee32f4
